### PR TITLE
TS: Remove 4 redundant definitions inherited from Curve

### PR DESCRIPTION
**Pull request by marquizzo**
CatmullRomCurve3.getPoint() should accept an [optional Vector3 target](https://github.com/mrdoob/three.js/blob/master/src/extras/curves/CatmullRomCurve3.js#L104) argument. It's missing in the TypeScript declaration file, which leads to compiler errors if users try to use this option.

### CODE REVIEW

File: src/extras/curves/CatmullRomCurve3.d.ts [hunk 1]
```diff
@@ -43,6 +43,6 @@ export class CatmullRomCurve3 extends Curve<Vector3> {
 
 	points: Vector3[];
 
-	getPoint( t: number ): Vector3;
+	getPoint( t: number, optionalTarget?: Vector3 ): Vector3;
```
Comment (Mugen87) [inline src/extras/curves/CatmullRomCurve3.d.ts]: How about deleting the entire `getPoint()` definition? `CatmullRomCurve3` is derived from `Curve` where `getPoint()` is already defined.

Comment (marquizzo) [reply src/extras/curves/CatmullRomCurve3.d.ts]: Yeah, that would also work. I assumed there was a reason why someone redefined this method, so I didn't feel comfortable just deleting it. But if you think that's best, I can remove the line altogether.

Comment (Mugen87) [reply src/extras/curves/CatmullRomCurve3.d.ts]: I think the definition is redundant. This is also true for `CubicBezierCurve3`,  `LineCurve3` and `QuadraticBezierCurve3`. Not sure why these methods are still there.

Comment (marquizzo) [reply src/extras/curves/CatmullRomCurve3.d.ts]: Ok, I can modify those files tonight. Stay tuned...

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**